utils/deploy_to_dev_env.sh

```diff
@@ -4,55 +4,107 @@
 # This can be achieved by using aws-vault and an amin role for example
 # aws-vault exec di-ipv-dev -- ./deploy_to_dev_env.sh
 
-if ! which jq >/dev/null; then
-  echo "Please install jq to use this script"
-fi
+declare -a stack_status=("UPDATE_COMPLETE", "UPDATE_ROLLBACK_COMPLETE")
+declare -a environments=("dev-danw" "dev-danp" "dev-danh" "dev-chrisw" "dev-amrits" "dev-kerrr" "dev-lait" "dev-ianj" "dev-derrenw" "dev-patrickb" "dev-vamh" "dev-wojciecho" "dev-aprilm")
```

Comment:
keeping this up to date will become an onboarding/off boarding exercise that I very much want to avoid. If the stack doesn't exist then report an error to the user. We don't need to validate the env at the cost of keeping this list up to date.